src/main/scripts/import_objects.groovy

```diff
@@ -189,6 +189,7 @@ else {
     }
 
     if (foundErrors) {
+		script.delete()
```

Comment:
I will make that change, good suggestion. Currently, there is a "script.delete()" on 130 that removed that input script, but is dependent on success, so added it to the failure also. I will add the  "script.deleteOnExit()" and remove both instances of the "script.delete()".